src/main/java/tc/oc/pgm/commands/RotationCommands.java

```diff
@@ -0,0 +1,247 @@
+package tc.oc.pgm.commands;
+
+import app.ashcon.intake.Command;
+import app.ashcon.intake.CommandException;
+import app.ashcon.intake.parametric.annotation.Default;
+import java.util.List;
+import net.md_5.bungee.api.ChatColor;
+import org.bukkit.command.CommandSender;
+import tc.oc.pgm.AllTranslations;
+import tc.oc.pgm.api.Permissions;
+import tc.oc.pgm.api.chat.Audience;
+import tc.oc.pgm.api.match.MatchManager;
+import tc.oc.pgm.map.PGMMap;
+import tc.oc.pgm.rotation.Rotation;
+import tc.oc.pgm.util.PrettyPaginatedResult;
+
+public class RotationCommands {
+  @Command(
+      aliases = {"rotation", "rot", "rt"},
+      desc = "Shows the maps in the active rotation",
+      usage = "[page]",
+      help = "Shows all the maps that are currently in the active rotation.")
+  public static void rotation(
+      Audience audience, CommandSender sender, MatchManager matchManager, @Default("1") int page)
+      throws CommandException {
+
+    List<PGMMap> maps;
+    Rotation rotation = matchManager.getRotationManager().getActiveRotation();
+    if (rotation != null) maps = rotation.getMaps();
+    else {
+      sender.sendMessage(
+          ChatColor.RED + AllTranslations.get().translate("command.rotation.noRotation", sender));
+      return;
+    }
+
+    int resultsPerPage = 8;
+    int pages = (maps.size() + resultsPerPage - 1) / resultsPerPage;
+
+    String listHeader =
+        ChatColor.BLUE.toString()
+            + ChatColor.STRIKETHROUGH
+            + "-----------"
+            + ChatColor.RESET
+            + " "
+            + AllTranslations.get().translate("command.rotation.currentRotation.title", sender)
+            + ChatColor.DARK_AQUA
+            + " ("
+            + ChatColor.AQUA
+            + rotation.getName()
+            + ChatColor.DARK_AQUA
+            + ")"
+            + " ("
+            + ChatColor.AQUA
+            + page
+            + ChatColor.DARK_AQUA
+            + " of "
```

Comment:
Should be translatable.

Reply:
Should I just plain create a translatable string for "of"? It wasn't translated on the MapCommands either, so I figured I'd just follow it's guide. https://github.com/Electroid/PGM/blob/ae0ad4750e80ce6266da1d914fdc89e69ad61283/src/main/java/tc/oc/pgm/commands/MapCommands.java#L94

Reply:
Sure, that's fine for now.